.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md/
```

Comment:
- Though you have included the `test.md` file in your `.gitignore` file, I can still see the file present in your repo 😭. This is because you haven't ignored it correctly. Please remove the `/` after the "test.md" only. The `/` is only needed when you want to ignore a directory/folder 😄 🙏